cluster/prod/README.md

```diff
@@ -0,0 +1,16 @@
+# Add a new component to the release
+
+1. Create the component, such that no other components depend on it. (If it fails, you can debug without worrying about breaking production.)
+1. **PR**: Add the component to the BUILD file, and send a PR for this. (Example: https://github.com/GoogleCloudPlatform/testgrid/pull/822).
+1. Wait until Container Registry has a date-versioned image for the component.
+    - e.g. https://github.com/GoogleCloudPlatform/testgrid/pull/822 -> tabulator image with version`v20220210-v0.0.121-16-g5bceb0c`
+1. **PR**: Create configuration .yaml files under cluster/canary and cluster/prod, referencing the image version all components are using in production (so the autobumper can catch it).
```

Comment:
I'd consider splitting these into two PRs and only merging the prod PR after the canary one works.